packages/front-end/services/features.ts

```diff
@@ -204,13 +209,21 @@ export function useFeaturesList(withProject = true, includeArchived = false) {
     hasArchived: boolean;
   }>(url);
 
+  useEffect(() => {
+    if (data) {
+      setFeatures(data.features);
+      setExperiments(data.linkedExperiments);
+      setHasArchived(data.hasArchived);
+    }
+  }, [data]);
```

Comment:
I don't think I need to have an `else` to reset the fields back to `[]` if the data changes back to empty, but could be wrong